packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -710,5 +712,23 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
         }
       }
     },
+  } as Plugin
+  if (enableNativePlugin) {
+    delete jsPlugin.transform
+    delete jsPlugin.resolveId
+    delete jsPlugin.load
   }
+  return [
+    jsPlugin,
+    enableNativePlugin
+      ? nativeBuildImportAnalysisPlugin({
```

Comment:
The `nativeImportAnalysisPlugin` only implemented `transform` `resolveId` and `load` using a hybrid plugin to make the function work as before.